.github/workflows/gradle.yml

```diff
@@ -0,0 +1,31 @@
+name: Java CI
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+      - refs/tags/*
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build
+        run: ./gradlew check
+      - name: Release
+        env:
+          BINTRAY_USER: jenkins-reform-hmcts
+          BINTRAY_KEY: ${{ secrets.BINTRAY_KEY }}
```

Comment:
this secret needs to be in project settings. team owners should confirm this is done and correct creds

Reply:
it's been done